src/Message/PurchaseRequest.php

```diff
@@ -140,14 +140,16 @@ protected function getEndPoint($postdata)
      * Generate the hmac hash to be passed in endpoint url
      *
      * Code modified from
-     * @link http://www.paystation.co.nz/cms_show_download.php?id=69
+     * @link https://docs.paystation.co.nz/#hmac-authentication
      * @return string hmac
      */
     protected function getHmac($timestamp, $postdata)
     {
         $authenticationKey = $this->getHmacKey();
         $hmacWebserviceName = 'paystation'; //webservice identification.
-        $hmacBody = pack('a*', $timestamp).pack('a*', $hmacWebserviceName).pack('a*', $postdata);
+        $hmacBody = pack('a*', $timestamp).
+            pack('a*', $hmacWebserviceName).
+            pack('a*', $postdata);
```

Comment:
Why drop onto new lines? Doesn't seem to add to readability?

Reply:
I thought it did, but don't hold the opinion strongly

Reply:
🤷‍♂ I think it's probably a short enough line (doesn't come anywhere near the soft character limit) that it probably doesn't need the line breaks.

Reply:
All good. Reverting.